### DISCUSSION TITLE
Addition of "hacktoberfest" topic to this repo

### DISCUSSION THREAD

**Discussion by kishaningithub** [aws-powertools/powertools-lambda-python#1578] [Q&A] [answered]
In my opinion this repo would attract a lot if new contributors if you can add "hacktoberfest" topic to this repository

More info - https://hacktoberfest.com/participation/#pr-mr-details

**Comment by heitorlessa** [comment, marked as answer]
hey @kishaningithub thanks for the heads up! We already had the required label (`hacktoberfest-accepted`) but I've just updated the topics with `hacktoberfest`. 

Thank you!
